src/abstractions/fileUpload.service.ts

```diff
@@ -0,0 +1,7 @@
+import { CipherString } from '../models/domain';
+import { SendFileUploadDataResponse } from '../models/response/sendFileUploadDataResponse';
+
+export abstract class FileUploadService {
```

Comment:
This abstraction is missing from `abstractions/index.ts`.

don't forget the `services.module.ts` in downstream repos as well (although not relevant here)

Reply:
There is a comment in index.ts informing its deprecation -- I believe we're just importing direct from files now rather than a one-stop-shop

https://github.com/bitwarden/jslib/blob/master/src/abstractions/index.ts#L1

Reply:
🤦🏻 , you're right, good catch, thanks.